npc/cities/amatsu.txt

```diff
@@ -48,403 +48,403 @@
 
 //= Amatsu Transportation ==================================
 -	script	::Sea_Captain_amatsu	FAKE_NPC,{
-	mes "[Walter Moers]";
-	mes "Hey, there.";
-	mes "Have you ever heard that there";
-	mes "are totally different countries";
-	mes "than the Rune-Midgard Kingdom?";
+	mes("[Walter Moers]");
```

Comment:
better not split big sentenses to many messages, but split parts in one message by ``\r``

this will allow translate whole sentense, and still will show many messages in clients